playwright/auth.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable */
+
+const { request } = require("@playwright/test");
+const dotenv = require("dotenv");
+const process = require("process");
+const crypto = require("crypto");
+const path = require("path");
+const fs = require("fs");
+
+dotenv.config();
+
+const PERMISSIONS = [
+  "admin",
+  "app",
+  "discount",
+  "order",
+  "channel",
+  "customer",
+  "giftCard",
+  "page",
+  "plugin",
+  "productTypeAndAttribute",
+  "product",
+  "settings",
+  "staff",
+  "shipping",
+  "translations",
+]
+
+const ACCOUNT_EMAILS = {
+  channel: "channel.manager@example.com",
+  shipping: "shipping.manager@example.com",
+  giftCard: "gift.card.manager@example.com",
+  app: "app.manager@example.com",
+  settings: "setting.manager@example.com",
+  page: "page.manager@example.com",
+  order: "order.manager@example.com",
+  translations: "translation.manager@example.com",
+  staff: "staff.manager@example.com",
+  customer: "user.manager@example.com",
+  productTypeAndAttribute: "product.type.and.attribute.manager@example.com",
+  discount: "discount.manager@example.com",
+  plugin: "plugin.manager@example.com",
+  product: "product.manager@example.com",
+};
+
+const createQuery = (email, password) => `mutation TokenAuth{
+  tokenCreate(email: "${email}", password: "${password}") {
+    token
+    refreshToken
+    errors: errors {
```

Comment:
question: do you need this repetition?
```suggestion
    errors {
```